Assets/Magic/Mana.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Mana : MonoBehaviour
+{
+    [SerializeField] private float maxMana;
```

Comment:
```suggestion
{
    [Header("Event Variables")]
    [SerializeField] private UnityEvent onFull= new UnityEvent();
    [SerializeField] private UnityEvent onEmpty = new UnityEvent();
    
    [SerializeField] private float maxMana;
```

Misshien deze unity events er in krijgen. Kan handig zijn voor later.

Reply:
UnityEvents Toegevoegd in [69a2582](https://github.com/Team-Swamp/Adley-s_Adventure/pull/3/commits/69a258275ee66e53690c6a989e3923e8396a257a)